### PR TITLE
Create Ayatt.net.xml

### PR DESCRIPTION
**Pull request by Giltyhub**

### CODE REVIEW

**Comment by pipboy96** [comment]
@Giltyhub
```
mail.ayatt.net - Found open ports: 443, 80
autodiscover.ayatt.net - Found open ports: 443, 80
webmail.ayatt.net - Found open ports: 443, 80
cpanel.ayatt.net - Found open ports: 443, 80
www.ayatt.net - Found open ports: 443, 80
webdisk.ayatt.net - Found open ports: 443, 80
```

File: src/chrome/content/rules/Ayatt.net.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="Ayatt.net">
+	<target host="ayatt.net" />
+	<target host="www.ayatt.net" />
```
Comment (pipboy96) [inline src/chrome/content/rules/Ayatt.net.xml]:
```diff
+	<target host="autodiscover.ayatt.net" />
+	<target host="cpanel.ayatt.net" />
+	<target host="mail.ayatt.net" />
+	<target host="webdisk.ayatt.net" />
+	<target host="webmail.ayatt.net" />
```

Comment (Giltyhub) [reply src/chrome/content/rules/Ayatt.net.xml]: thank u so much @pipboy96

**Comment by pipboy96** [comment]
No subdomains including `^` resolve.

**Pull request closed without merging**